qemu/tests/cfg/invalid_cpu_device_hotplug.cfg

```diff
@@ -24,9 +24,7 @@
             ppc64, ppc64le:
                 error_desc = "core {0} already populated"
             x86_64:
-                Windows:
-                    # TBD: Please update the list when new supported Win OS comes out
-                    only Win2008, Win2012, Win2016, Win2019
+                no WinXP WinVista Win2000 Win2003 Win7 Win8 Win10
```

Comment:
what's the expect result for 32bit ?

Reply:
They are not officially supported for these features. So we don't test it.

Reply:
> They are not officially supported for these features. So we don't test it.

So, it is an existing bug that this case can't prevent 32bit guest,  how do you think about update it together ~